Modules/Synchronization/src/Synchronization.Infrastructure.Database.Postgres/Migrations/20231004081756_AddBlobReferencePropertyToDatawalletModification.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace Backbone.Modules.Synchronization.Infrastructure.Database.Postgres.Migrations
+{
+    /// <inheritdoc />
+    public partial class AddBlobReferencePropertyToDatawalletModification : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<string>(
+                name: "BlobReference",
+                table: "DatawalletModifications",
+                type: "character(32)",
+                unicode: false,
+                fixedLength: true,
+                maxLength: 32,
+                nullable: false,
```

Comment:
Shouldn't it be nullable?

Reply:
No. Because new modifications should always specify a value.

New modifications will receive the default value (=> an empty string)

Reply:
Did you mean to write "Old modifications will receive the default value (=> an empty string)" in your comment?
If yes, then I understand (y)

Reply:
Ahm, yeah, of course I did. ;-)